artifacts/CAMARA_common.yaml

```diff
@@ -1,480 +1,480 @@
 openapi: 3.0.3
 info:
- title: Camara common data types
- description: Common data types for Camara APIs
- contact:
-  email: sp-com@lists.camaraproject.org
- license:
-  name: Apache 2.0
-  url: https://www.apache.org/licenses/LICENSE-2.0.html
- version: 0.4.0-wip
+  title: Camara common data types
+  description: Common data types for Camara APIs
+  contact:
+    email: sp-com@lists.camaraproject.org
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.4.0-wip

```

Comment:
Should we take advantage and set as "version: 0.4.0", as the stable one for MetaRelease v0.4.0